src/client/Microsoft.Identity.Client/Microsoft.Identity.Client.csproj

```diff
@@ -170,6 +179,9 @@
   </PropertyGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == '$(TargetFrameworkNetDesktop462)' ">
+    <Compile Include="$(PathToMsalSources)\Platforms\Features\OpenTelemetry\**\*.cs" />
+    
+    <PackageReference Include="System.Diagnostics.DiagnosticSource" Version="7.0.2" />
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == '$(TargetFrameworkNetDesktop45)' ">
```

Comment:
We bumped net461 to net462 to support OTEL, are there any special considerations for net45?

Reply:
Net 45 is excluded from logging any telemetry through otel. We did drop net core 2.1 and bump to net 462 but net 45 is still supported as there may be dependencies on it.